model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/JpaMapStorageProviderFactory.java

```diff
@@ -438,14 +434,7 @@ private void printOperationalInfo(Connection connection) {
 
     private void update(Class<?> modelType, Connection connection, KeycloakSession session) {
         KeycloakModelUtils.runJobInTransaction(session.getKeycloakSessionFactory(), (KeycloakSession lockSession) -> {
-            // TODO locking tables based on modelType: https://github.com/keycloak/keycloak/issues/9388
-            DBLockProvider dbLock = session.getProvider(DBLockProvider.class);
-            dbLock.waitForLock(DBLockProvider.Namespace.DATABASE);
-            try {
-                session.getProvider(MapJpaUpdaterProvider.class).update(modelType, connection, config.get("schema"));
-            } finally {
-                dbLock.releaseLock();
-            }
```

Comment:
What is the purpose of removing these lines? AFAICS, these are no-op in `none` lock. Wouldn't these be used once the proper locking is implemented in #9388?